### PR TITLE
Move generated api code to pkg package

### PR DESCRIPTION
**Pull request by alexmt**
PR move generated api code to pkg package. This allows using api client without introducing dependency to the api server implementation.

### CODE REVIEW

**Comment by jessesuen** [comment]
Instead of:
`github.com/argoproj/argo-cd/pkg/serverapis/account `

how about we put them under `apiclient`. e.g.:
`github.com/argoproj/argo-cd/pkg/apiclient/account`

**Review by jessesuen — APPROVED** [review]
Conditional approval based on agreement that we move under apiclient.

File: cmd/argocd/commands/account.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func NewAccountUpdatePasswordCommand(clientOpts *argocdclient.ClientOptions) *co
 				errors.CheckError(err)
 			}
 
-			updatePasswordRequest := account.UpdatePasswordRequest{
+			updatePasswordRequest := accountpkg.UpdatePasswordRequest{
```
Comment (alexec) [inline cmd/argocd/commands/account.go]: Rather than `accountpkg` what about `accountapi`?

**Review by alexec — APPROVED** [review]
One comment about package import naming.

**Comment by alexmt** [comment]
Moved packages to `apiclient`

**Pull request merged**